style.css

```diff
@@ -1 +1,227 @@
-/*  Add your styling here */
+.content{
+    background-color: skyblue;
+    display:grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: auto;  
+    grid-template-areas: 
+    "header header header header"
+    "main main main main" 
+    "footer footer footer footer";
+    
+}
+
+/* Header */
+.header{
+    grid-area: header;
+    background-color:rgb(122, 230, 167) ;
```

Comment:
Now you know about CSS variables - you make use of them to define the colors at the top of your file